### DISCUSSION TITLE
Tip for MacOS users: Show current file in Finder!

### DISCUSSION THREAD

**Discussion by cblc** [unixwork/xnedit#111] [General]
If you are using XNEdit on MacOS, let me tell you how you can get a nice feature for revealing in a Finder window the file currently edited: 

Add a new shell command (Preferences>Default Settings>Customize Menus>Shell Menu...) and create a new command called "Show in MacOS Finder", and assign a shortcut to it (I assigned it to F7 because it's the first free function key that I have). As Command Input, select "None". As Command Output, I selected "Dialog" (I don't really know if that's the proper choice, but works for me).

And then, on Shell Command to execute, type `open . ` (IMPORTANT: Don't forget the dot!!)

Create the new command, and you'll get it working (in your Shell menu, and also connected to the shortcut you specified).

Maybe your next wish would be to have not only "Show in Finder", but "Open Terminal Here" as well, but for that purpose I'm using one of those "Open Terminal Here" applets that you can drag to your Finder toolbar (there are some of them in GitHub), so if I have a Finder window, I already have a button on it for opening a new terminal on that folder.

As a reference here you have a screenshot --yes, I'm using SGI IndigoMagic-looking resources, I'm a hard-die IRIX fan 😄 

![Captura de pantalla 2023-07-01 a las 11 43 23](https://github.com/unixwork/xnedit/assets/53348558/c7800b51-85b3-44ee-8a08-9d51e962fe2f)

**Comment by unixwork** [comment]
To open a terminal you can use `open -a Terminal .`

And the IRIX resources look really nice.

**Reply by cblc** [reply]
True, it works perfectly!
